open_academy/models/session.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, fields
+
+
+class Session(models.Model):
+    _name = 'session'
+    _description = 'Session'
+
+    name = fields.Char(required=True)
+    start_date = fields.Date('Start date')
+    duration = fields.Float('Duration')
```

Comment:
If you are trying to add as string, Odoo automatically set the name field as `Duration` so we don't need replace to the same string

```suggestion
    duration = fields.Float()
```